.github/workflows/CI.yml

```diff
@@ -1090,4 +1146,9 @@ jobs:
       - name: clean up SSM test package
         run: |
           ssm_pkg_version=`cat build/packages/VERSION`
-          aws ssm delete-document --name "testAWSDistroOTel-Collector" --version-name ${ssm_pkg_version}
+          for region in $INTEGRATION_TEST_FIRST_REGION $INTEGRATION_TEST_SECOND_REGION
+          do
+              aws ssm describe-document --name ${SSM_PACKAGE_NAME} --version-name ${ssm_pkg_version} --region ${region} >/dev/null 2>&1 && \
```

Comment:
Why are you describing the document before deleting it? It'll always exist.

Reply:
It depends though. In some way if we had an error before the SSM was created, then we cannot delete what's not there. Got an error when testing with one of my PR.